app/include/user_modules.h

```diff
@@ -59,6 +60,7 @@
 #define LUA_USE_MODULES_SPI
 //#define LUA_USE_MODULES_STRUCT
 //#define LUA_USE_MODULES_SWITEC
+//#define LUA_USE_MODULES_TCA9535
```

Comment:
Interesting, I had missed that... 😃  @saucompeng that's one of the reasons why we recommend working with dedicated branches per feature rather than committing to `dev` in your fork.